packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -20,10 +20,21 @@ var _ = require('lodash');
 var rhosts = /github|bitbucket|gitlab/i;
 var rtag = /tag:\s*[v=]?(.+?)[,\)]/gi;
 
+function semverTagsPromise(options) {
+  var semverTagsDeferred = Q.defer();
```

Comment:
Has `conventional-changelog-core` already dropped node `<v4` support? Then it would be nice to promisify things.. Or do it anyway with a major bump :smile: Anyway, I agree, defer is not very nice here.